package.json

```diff
@@ -5,11 +5,14 @@
   "main": "index.js",
   "scripts": {
     "test": "./node_modules/mocha/bin/_mocha --timeout 20000 --recursive test/",
-    "start": "node -r dotenv/config bin/server.js"
+    "start": "node -r dotenv/config bin/server.js",
+    "lint": "./node_modules/.bin/eslint --ext .js .",
+    "audit": "npm audit"
```

Comment:
`run npm audit` instead of just using `npm audit`?

Reply:
`npm audit` is correct, I'm not sure why a script is being added though. It is not necessary.

Reply:
Correct. Technically, I did not have to add it as a script in package.json; however visually, I added so that we could see "audit" being supported. 😄

Reply:
In the script, "npm audit" is correct. When calling externally, the typed command should "npm run audit".